crds/workspace.devfile.io_devworkspaces.v1beta1.yaml

```diff
@@ -550,8 +550,8 @@ spec:
                           - dockerfile
                         properties:
                           autoBuild:
-                            description: Defines if the image should be built during
-                              startup Default is false
+                            description: "Defines if the image should be built during
+                              startup \n Default value is `false`"
```

Comment:
Sorry for the nitpick but I know these descriptions make it in the doc.  Can we end the sentence after the word `startup` with a "."

Reply:
updated